File: ws/pazmino/u2/WS15Inheritance/ZooApp/src/main/java/ec/edu/espe/zooapp/model/Vaccine.java
package ec.edu.espe.zooapp.model;

import java.util.ArrayList;
import java.util.Date;

/**
 *
 * @author Brandon Pazmino
 */
public class Vaccine {

    public Vaccine(int vaccineId, String vaccineName) {
    }
    
    int id;
    String description;
    String name;
    Date borOnDate;
    float weight;
    char sex;
    ArrayList<Vaccine> vaccines;

}

File: hw/robalino/u2/hw18inheritance/vaccine.py
class Vaccine:
    def __init__(self, id: int, description: str):
        self._id = id
        self._description = description

    def __str__(self) -> str:
        return f"Vaccine(id={self._id}, description={self._description})"

    @property
    def id(self) -> int:
        return self._id

    @id.setter
    def id(self, value: int):
        self._id = value

    @property
    def description(self) -> str:
        return self._description

    @description.setter
    def description(self, value: str):
        self._description = value
